runtime/queries/printf/highlights.scm

```diff
@@ -0,0 +1 @@
+(format) @character
```

Comment:
`@character.special`?

Reply:
I kinda want to differentiate format strings from other special characters.

Reply:
I'd like that too, but I feel character and string are somewhat oftenly used as the same color in colorschemes

A custom capture wouldn't be bad imo, like `@character.format`

Reply:
A custom capture would be pointless since `@character.printf` can be used anyway.

Reply:
Right, this is fine then